### PR TITLE
docs: Update PowerAI installation

### PR DESCRIPTION
**Pull request by rayjh**

### CODE REVIEW

File: docs/Running-paie.rst [hunk 1]
```diff
@@ -76,14 +78,10 @@ Installation of the PowerAI Enterprise software involves the following steps;
 Preparation of the client nodes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Insure that the set up steps on IBM Knowledge Center up through and including 'Mount a shared file system' have been completed.
-
-https://www.ibm.com/support/knowledgecenter/SSFHA8_1.1.0/enterprise/powerai_setup.html
-
-Record the following information;
+Before proceding, you will need to gather the following information;
```
Comment (jaywcarman) [inline docs/Running-paie.rst]: proceding -> proceeding

File: docs/Running-paie.rst [hunk 2]
```diff
@@ -76,14 +78,10 @@ Installation of the PowerAI Enterprise software involves the following steps;
...
 
 -  hostname for each client node
--  Userid and password or private ssh key for the client nodes. Note that for installation, the same user id and password must exist on all client nodes. The user id used for installation must be configured with sudo access.
+-  Userid and password or private ssh key filr for the client nodes. Note that for running automated installation, the same user id and password must exist on all client nodes and must be configured with sudo access.
```
Comment (jaywcarman) [inline docs/Running-paie.rst]: ... or private ssh key *pair* for the client ...
... that for running _an_ automated installation, ...

File: docs/Running-paie.rst (hunk 1 above)
Comment (rayjh) [reply docs/Running-paie.rst]: Done.

File: docs/Running-paie.rst (hunk 2 above)
Comment (rayjh) [reply docs/Running-paie.rst]: Done

**Pull request merged**